### PR TITLE
various lint fixes

### PR DESCRIPTION
**Pull request by devoncarew**
- various lint fixes

### CODE REVIEW

File: src/io/flutter/FlutterErrorReportSubmitter.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public boolean submit(@NotNull IdeaLoggingEvent[] events,
                         @NotNull Component parentComponent,
                         @NotNull Consumer<SubmittedReportInfo> consumer) {
     if (events.length == 0) {
-      fail(((Consumer<SubmittedReportInfo>)consumer));
```
Comment (stevemessick) [inline src/io/flutter/FlutterErrorReportSubmitter.java:59]: This is required to compile with current AS sources.

File: src/io/flutter/FlutterErrorReportSubmitter.java [hunk 2]
```diff
@@ -96,7 +96,7 @@ public boolean submit(@NotNull IdeaLoggingEvent[] events,
     final DataContext dataContext = DataManager.getInstance().getDataContext(parentComponent);
     final Project project = PROJECT.getData(dataContext);
     if (project == null) {
-      fail(((Consumer<SubmittedReportInfo>)consumer));
```
Comment (stevemessick) [inline src/io/flutter/FlutterErrorReportSubmitter.java:99]: This is required to compile with current AS sources.

File: src/io/flutter/FlutterErrorReportSubmitter.java [hunk 3]
```diff
@@ -189,7 +189,7 @@ public boolean submit(@NotNull IdeaLoggingEvent[] events,
     final VirtualFile file = scratchRoot.createScratchFile(project, "bug-report.md", PlainTextLanguage.INSTANCE, text);
 
     if (file == null) {
-      fail(((Consumer<SubmittedReportInfo>)consumer));
```
Comment (stevemessick) [inline src/io/flutter/FlutterErrorReportSubmitter.java:192]: This is required to compile with current AS sources.

**Review by stevemessick — CHANGES_REQUESTED** [review]
I found three changes that will cause compilation errors in the current Android Studio environment.

**Comment by devoncarew** [comment]
> I found three changes that will cause compilation errors in the current Android Studio environment.

Thanks for checking! I'll revert and add comments so I don't unintentionally re-add.

How are these passing in the CI though?

**Comment by stevemessick** [comment]
Thanks!

> How are these passing in the CI though?

They both use Java 11.

File: src/io/flutter/FlutterErrorReportSubmitter.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/FlutterErrorReportSubmitter.java:99]: Reverted (and the two places below as well).

**Comment by devoncarew** [comment]
> They both use Java 11.

Ah, gotcha.

I see that we're installing jdk11 on the bots (https://github.com/flutter/flutter-intellij/blob/master/.travis.yml#L6). It looks like travis has some capability to change between jdk versions in the middle of a run:

https://docs.travis-ci.com/user/languages/java/#switching-jdks-java-8-and-below-within-one-job

**Review by stevemessick — APPROVED** [review]

**Pull request merged**